rts/lra/lra-annotations/src/main/java/org/jboss/narayana/rts/lra/annotation/CompensatorStatus.java

```diff
@@ -0,0 +1,34 @@
+package org.jboss.narayana.rts.lra.annotation;
+
+/**
+ * The status of a compensator. The status is only valid after the coordinator has told the compensator to
+ * complete or compensate. The name value of the enum should be returned by compensator methods marked with
+ * the {@link Status} annotation.
+ */
+public enum CompensatorStatus {
+    /**
+     * the Compensator is currently compensating for the LRA
+     */
+    Compensating,
+    /**
+     * the Compensator has successfully compensated for the LRA
+     */
+    Compensated,
+    /**
+     * the Compensator was not able to compensate for the LRA (and must remember
+     * it could not compensate until such time that it receives a forget message)
+     */
+    FailedToCompensate,
```

Comment:
This is never used - I guess that means we need to add a test that uses it

Reply:
It does, I have added it to my TODO list